lightning/src/ln/channelmanager.rs

```diff
@@ -119,6 +121,12 @@ pub(super) struct HTLCPreviousHopData {
 	incoming_packet_shared_secret: [u8; 32],
 }
 
+struct ClaimableHTLC {
+	src: HTLCPreviousHopData,
```

Comment:
s/src/prev_hop ? otherwise I might assume that `src` refers to the original sender

Reply:
Sure. Though note that, in general, we *never* know the actual source of an HTLC, so its somewhat moot.